service/static/css/muk.institution.min.css

```diff
@@ -1 +1,405 @@
-results-download{text-align:right}.edges-bs3-searching-notification-bar{width:100%;text-align:center;color:#fff}.edges-bs3-or-term-selector-filter-remove,.edges-bs3-or-term-selector-filter-remove:hover,.edges-bs3-or-term-selector-header a{color:#000}.edges-bs3-or-term-selector-facet{border-top:1px solid #ccc;border-bottom:1px solid #ccc}.edges-bs3-or-term-selector-header{font-weight:700;padding-left:5px;padding-top:5px;padding-bottom:5px;background-color:#eee}.edges-bs3-or-term-selector-result{padding-left:5px;padding-top:2px;padding-bottom:2px}.edges-bs3-or-term-selector-result:nth-of-type(odd){background-color:#fff}.edges-bs3-or-term-selector-result:nth-of-type(even){background-color:#f4f4f4}.edges-bs3-or-term-selector-result:hover{background-color:#eee}.edges-bs3-or-term-selector-selections::-webkit-scrollbar{-webkit-appearance:none}.edges-bs3-or-term-selector-selections::-webkit-scrollbar:vertical{width:8px}.edges-bs3-or-term-selector-selections::-webkit-scrollbar:horizontal{height:8px}.edges-bs3-or-term-selector-selections::-webkit-scrollbar-thumb{border-radius:8px;border:1px solid #c4d0d7;background-color:rgba(0,0,0,.4)}.edges-bs3-or-term-selector-selections::-webkit-scrollbar-track{background-color:#c4d0d7;border-radius:8px}#date_range{float:right;margin-bottom:20px}.ui-datepicker-year{background:#f6a828}.muk-institution-report-template-filters{padding:10px;background:#eee;margin-bottom:20px}.muk-institution-report-template-filter-header{margin-bottom:12px;font-size:.9em;letter-spacing:3px;color:#9a9a9a;-webkit-font-smoothing:antialiased;text-shadow:1px 1px 1px rgba(0,0,0,.004)}.muk-institution-report-template-filter{font-family:'Stag Sans Web Light',HelveticaNeue-Light,'Helvetica Neue Light','Helvetica Neue',Arial,sans-serif;margin-top:12px;margin-bottom:20px}.edges-bs3-n-separate-or-term-selector-container{margin-top:20px;margin-bottom:20px}.muk-institution-report-template-tab-bar{border-bottom:2px solid #eee;text-align:right;line-height:44px;color:#999}.muk-institution-report-template-tab-bar li{margin-bottom:-2px}.muk-institution-report-template-tab-bar li.active>a,.muk-institution-report-template-tab-bar li.active>a:focus,.muk-institution-report-template-tab-bar li.active>a:hover{border-left:2px solid #eee;border-right:2px solid #eee;border-top:2px solid #0e90d2;border-bottom:3px solid #fff}.muk-institution-report-template-tab-container{width:100%;height:auto}.muk-institution-report-template-tab-container .muk-institution-report-template-tab{height:700px}.edges-bs3-tabular-results-table{border-collapse:collapse;text-align:right}.edges-bs3-tabular-results-cell-Institution{text-align:left;white-space:nowrap}#date_range{padding:10px;background:#eee}.edges-bs3-bs-multi-date-range-select{margin:10px 0}.edges-bs3-tabular-results-tablediv{max-height:600px;overflow-y:scroll}#edges-bs3-tabular-results-download-data_table{text-decoration:underline}.edges-bs3-tabular-results-table td,.edges-bs3-tabular-results-table th{border:1px solid #e4e4e4}
```

Comment:
Well check for yourself...https://github.com/JiscMonitor/monitor-uk/tree/a7cedd0d56c220f6475507be7aa5827713e1da9d/service/static/css
I've reverted everything to it's original state